agent_application/docker/README.md

```diff
@@ -12,14 +12,23 @@ docker build -f docker/Dockerfile -t ssi-agent ..
 
 Inside the folder `/agent_application/docker`:
 
-1. _Inside `docker-compose.yml` replace the value `<your-local-ip>` for the environment variable `AGENT_APPLICATION_HOST` with your actual local ip address (such as 192.168.1.234)_
+1. _Inside `docker-compose.yml` replace the environment value: `AGENT_APPLICATION_URL` with your actual local ip address (such as http://192.168.1.234:3033)
 2. Optionally, add the following environment variables:
     - `AGENT_ISSUANCE_CREDENTIAL_NAME`: To set the name of the credentials that will be issued.
     - `AGENT_ISSUANCE_CREDENTIAL_LOGO_URL`: To set the URL of the logo that will be used in the credentials.
 3. To start the **SSI Agent**, a **Postgres** database along with **pgadmin** (Postgres Admin Interface) simply run:
 
 ```bash
-docker compose up -d
+docker compose up
```

Comment:
Why remove the `-d`?

Reply:
Because when developing i want to restart whenever I make changes, and I want to see the logs. Why would you want to run it in the background? then you can't quit easily